### PR TITLE
Feature/CI test and how to test

### PR DESCRIPTION
**Pull request by laysabit**
<details>
  <summary>PR Checklist</summary>

### PR Structure

- [X] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
- [X] This PR avoids mixing refactoring changes with feature changes (split into two PRs
      otherwise).

### Thoroughness

- [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
- [X] I've updated the docs and README with the added features, breaking changes, new instructions on how to use the repository.

### Release planning

- [X] I've decided if this PR requires a new major/minor/patch version accordingly to
    [semver](https://semver.org/), and I've changed the name of the BRANCH to release/* , feature/* or patch/* .
</details>

### What

README instructions on how to start testing in the repository.

### Why

Without it, if code is send to production without being testing, production will break.

### Known limitations

[N/A]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,33 @@ dbt_utils package. Read more about them [here](https://hub.getdbt.com/dbt-labs/d
...
+            catalog: "{{ env_var('DEV_CATALOG_NAME')}}"
+            schema: "{{ env_var('DEV_SCHEMA_NAME')}}"
+            host: "{{ env_var('DEV_HOST') }}"
+            http_path: "{{ env_var('DEV_HTTP_PATH') }}"
+            token: "{{ env_var('DEV_TOKEN') }}"
+            threads: 16
+            ansi_mode: false
+```
+
+When it is done, there are two necessary commands for working locally without difficulties:
```
Comment (vitoravancini) [inline README.md:56]: Why do we need this commands? 
I dont remeber running thoses

File: README.md [hunk 2]
```diff
@@ -34,6 +34,33 @@ dbt_utils package. Read more about them [here](https://hub.getdbt.com/dbt-labs/d
 
 This package works for most of EL processes and depends on the metadata generated by the respective platform.
 
+## Profiles
+
```
Comment (vitoravancini) [inline README.md:38]: We should make clear that this profile example is for running in databricks. For running in snowflake or other target users should read the connectors specific profiles.
We dont need one example for each case, just make this point clear

File: README.md (hunk 1 above)
Comment (laysabit) [reply README.md:56]: It is best to work using a .env file that can easily be edited, if necessary, directly in the repository, than creating a ~/.dbt with the databrick's values. It is easier to deal with. The setup.sh related commands are to facilitate the user's life, so the person does not have to create an env and configure it on their own. It also do the dbt deps for the user, so it is less commands to do manually

File: README.md (hunk 2 above)
Comment (laysabit) [reply README.md]: Done

**Review by vitoravancini — APPROVED** [review]

**Pull request merged**